Standards/scs-0217-v1-cluster-hardening.md

```diff
@@ -0,0 +1,470 @@
+different Kubelets by disallowing modification of non-bound.
+
+At last, *Pod security standards* in the form of policies MUST be activated for the cluster. The SCS REQUIRES at least
+the *Baseline* policy with the *Restricted* policy CAN also be used.
+The *Pod security* admission controller MUST also be activated in order to enforce these policies on a namespace level.
+We RECOMMEND the `enforce` level to be used for this admission controller setup.
+
+## Conformance Tests
+
+Conformance Tests will be written within another issue.
+
+## Related Documents
+
+[OWASP Kubernetes Security Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Kubernetes_Security_Cheat_Sheet.html)
+[Kubernetes security concepts](https://kubernetes.io/docs/concepts/security/)
+[Securing a cluster](https://kubernetes.io/docs/tasks/administer-cluster/securing-a-cluster/)
+[Controlling access](https://kubernetes.io/docs/concepts/security/controlling-access/)
+[Pod security standards](https://kubernetes.io/docs/concepts/security/pod-security-standards/)
+[NSA CISA Kubernetes hardening](https://kubernetes.io/blog/2021/10/05/nsa-cisa-kubernetes-hardening-guidance/)
+[Configure etcd](https://kubernetes.io/docs/tasks/administer-cluster/configure-upgrade-etcd/)
+[Google Kubernetes cluster trust](https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-trust)
+
+[kubernetes-releases]: https://kubernetes.io/releases/
+[kubernetes-version-skew]: https://kubernetes.io/releases/version-skew-policy/
+[strong-credentials]: https://pages.nist.gov/800-63-3/sp800-63b.html
+[kubernetes-auth]: https://kubernetes.io/docs/reference/access-authn-authz/authentication/
+[node-authorization]: https://kubernetes.io/docs/reference/access-authn-authz/node/
+[node-restriction]: https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction
+[kubelet-auth]: https://kubernetes.io/docs/reference/access-authn-authz/kubelet-authn-authz/#kubelet-authorization
+[pod-security-standards]: https://kubernetes.io/docs/concepts/security/pod-security-standards/
+[openidconnect]: https://kubernetes.io/docs/reference/access-authn-authz/authentication/#openid-connect-tokens
+[webhook-token]: https://kubernetes.io/docs/reference/access-authn-authz/authentication/#webhook-token-authentication
+[authenticating-proxy]: https://kubernetes.io/docs/reference/access-authn-authz/authentication/#authenticating-proxy
+
+[ro-port-removal]: https://github.com/kubernetes/kubernetes/issues/12968
+[ro-port-disabled]: https://github.com/kubernetes/kubernetes/pull/59666
+[ro-port-disabled-kubeadm]: https://github.com/kubernetes/kubeadm/issues/732
+[ro-port-s1]: https://www.stigviewer.com/stig/kubernetes/2021-04-14/finding/V-242387
+[ro-port-s2]: https://docs.datadoghq.com/security/default_rules/cis-kubernetes-1.5.1-4.2.4/
+[nsa-cisa]: https://kubernetes.io/blog/2021/10/05/nsa-cisa-kubernetes-hardening-guidance/
+[etcd-auth]: https://etcd.io/docs/v3.3/op-guide/authentication/
```

Comment:
we should probably include a linter in our pipeline which can check if all these links still work, else we will suffer from link rot in the future.

Reply:
Link rot won't happen if we use permalinks (/sarcasm)

Reply:
Im gonna resolve this, the previously mentioned issue will be worked on by @martinmo